nav2_core/include/nav2_core/recovery.hpp

```diff
@@ -42,8 +46,8 @@ class Recovery
    * @param  costmap_ros A pointer to the costmap
    */
   virtual void configure(
-    const nav2_util::LifecycleNode * parent,
-    const std::string & name, tf2_ros::Buffer * tf) = 0;
+    const nav2_util::LifecycleNode::SharedPtr parent,
```

Comment:
I think we've been using `rclcpp_lifecycle::LifecycleNode::SharedPtr`

Reply:
Can do, didn't notice that, wasn't intentional